tests/filecheck/transforms/convert_onnx_to_linalg.mlir

```diff
@@ -64,6 +64,15 @@
 // CHECK-NEXT:  %res_gemm_2_1 = linalg.matmul ins(%8, %t12 : tensor<5x10xf32>, tensor<10x3xf32>) outs(%res_gemm_2 : tensor<5x3xf32>) -> tensor<5x3xf32>
 // CHECK-NEXT:  %res_gemm_2_2 = linalg.add ins(%res_gemm_2_1, %10 : tensor<5x3xf32>, tensor<5x3xf32>) outs(%res_gemm_2_1 : tensor<5x3xf32>) -> tensor<5x3xf32>
 
+%t26 = "test.op"(): () ->  (tensor<1x16x14x14xf32>)
+%res_max_pool_single_out = "onnx.MaxPoolSingleOut"(%t26) {onnx_node_name = "/MaxPoolSingleOut", "auto_pad" = "NOTSET", "ceil_mode" = 0 : si64, "kernel_shape" = [3 : i64, 3 : i64],  "dilations" = [1 : i64, 1 : i64],  "pads" = [0 : i64, 0 : i64, 0 : i64, 0 : i64],  "storage_order" = 0 : si64, strides = [3 : i64, 3 : i64]} : (tensor<1x16x14x14xf32>) -> tensor<1x16x4x4xf32>
+
+// CHECK-NEXT:   %t26 = "test.op"() : () -> tensor<1x16x14x14xf32>
+// CHECK-NEXT:  %res_max_pool_single_out = tensor.empty() : tensor<3x3xf32>
+// CHECK-NEXT:  %res_max_pool_single_out_1 = tensor.empty() : tensor<1x16x4x4xf32>
+// CHECK-NEXT:  %res_max_pool_single_out_2 = arith.constant 1.844224e+19 : f64
```

Comment:
as you can see, this is incorrect, unfortunately

Reply:
I wonder if passing `float('-inf')` would work here

Reply:
` %res_max_pool_single_out_2 = arith.constant -inf : f64`
i think it does

Reply:
actually erm

Reply:
Can you please check that it works with MLIR?

Reply:
https://peps.python.org/pep-0754/

so -1e400 is equivalent to float('-inf') so can't it be sufficent to but that or just under that

Reply:
> https://peps.python.org/pep-0754/
> 
> so -1e400 is equivalent to float('-inf') so can't it be sufficent to but that or just under that

the sys max is -1e308, so anything higher returns -inf

Reply:
@superlopuh is this okay?

Reply:
I'd say this is still a little broken but at this point it's not a blocker. Could you please file an issue for this, and add it in a comment so readers aren't confused if they come across it in the future? Something like:
"dialects: arith Add a way to represent +-inf arith.constant in xDSL that roundtrips with MLIR for f32 and f64"

Reply:
will do, for sure :)